build.gradle

```diff
@@ -9,7 +9,7 @@ plugins {
 }
 
 group 'uk.gov.hmcts.reform'
-version '0.1.0'
+version '0.1.1'
```

Comment:
Would you mind changing the version to `0.2.0`? This change looks like neither a bugfix nor a minor modification.